core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -374,6 +375,7 @@ object KafkaConfig {
   val BrokerSessionTimeoutMsProp = "broker.session.timeout.ms"
   val NodeIdProp = "node.id"
   val MetadataLogDirProp = "metadata.log.dir"
+  val MetadataSnapshotMinNewRecordBytesProp = "metadata.log.snapshot.min.new_record.bytes"
```

Comment:
Hmm, I think we should avoid underscores in configuration key names.

Also, min.new_record.bytes is a bit unclear (to me at least).

How about `metadata.log.min.record.bytes.between.snapshots` ?

Reply:
I took your suggestion.